.github/workflows/mdlint.yml

```diff
@@ -0,0 +1,20 @@
+name: markdownlint
+on:
+  push:
+    paths:
+      - '**/*.md'
+      - '.markdownlint.json'
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  mdlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: markdownlint-cli
+        uses: nosborn/github-action-markdown-cli@v1.1.1
+        with:
+          files: '**/*.md'
+          ignore_path: ".markdownlintignore"
```

Comment:
This is not supported by v1.1.1
Deleting the line is fine as we're using the default filename anyway.
You may also bump the action to `@v3` tag instead of `v1.1.1`, as you did before.

Reply:
Now papers/2024/GPAM/README.md:59 is not being linted :/